PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Pull request title
+
+<!-- Open this PR as draft while it is not ready -->
+<!-- Use keywords like 'close' or 'solves' to link this PR to an issue -->
+
+This PR solves #12345
+
+#### Pull request checklist
+
+<!-- Before submitting the PR, please address each item -->
+
+- [ ] **Tests**: This PR includes tests for covering the features or bug fixes, or it includes an explanation of why it does not.
+- [ ] **Screenshots and/or logs**: This PR includes screenshots, videos or logs of the changes made, or an explanation of why it does not.
+- [ ] **Commits descriptions**: All commits in this PR contain a title and description according to [CONTRIBUTING.md](https://github.com/profusion/quickstart-nodejs-rest/blob/master/doc/CONTRIBUTING.md#commits).
+
+<!-- Also, don't forget to review your code before marking it as ready to merge -->
+
+### Progress
+
+<!-- If your PR is WIP, use checkboxes to show that you did and what you have to do. For example: -->
+
+- [x] New endpoint created;
+- [ ] Update organizations;
+- [ ] Create tests;
+
+### Description
+
+<!-- Describe your solution for the issue here. Adding videos, screenshots and expected logs may help the others to understand what is new.
+-->
+
+### How to test it
+
+<!-- Describe how the reviewers can test your feature. -->
```

Comment:
Missing a `Visual Reference` section at the end:
```markdown
## Visual reference

<!--
Please include screenshots, gifs or recordings
For example: if this is a bug fix, provide before and after screenshots

<img width="350" alt="Screenshot of bug fix" src="your-image-url-here">

Before | After
:-:|:-:
<img width="350" alt="Screenshot of screen pre bug fix" src="your-image-url-here"> | <img width="350" alt="Screenshot of screen post bug fix" src="your-image-url-here">
-->
```

Reply:
Visual reference section added at the end.